src/callgrind.rs

```diff
@@ -4,38 +4,62 @@ use std::io;
 
 use initialize::StackFrame;
 
+// Stats about the relationship between two functions, one of which
+// calls the other.
 #[derive(Debug)]
 struct Call {
+    // How many times has the outer function called the inner one?
     count: usize,
+
+    // How many samples were found inside the inner function
+    // (including all sub-functions), when called by the outer?
     inclusive: usize,
 }
 
+// Stats about a single function.
 #[derive(Debug, Default)]
 struct Location {
+    // How many samples were found inside this function only, not
+    // including calls to sub-functions?
     exclusive: usize,
```

Comment:
does this `exclusive` field have the same value as the `exclusive` field on StackEntry for the corresponding function?